### DISCUSSION TITLE
AutoRA Workflow Management Tools available in beta

### DISCUSSION THREAD

**Discussion by hollandjg** [AutoResearch/autora#387] [Show and Tell]
Hi all, the aurora-workflow package is available! 

Installation and how-to guides for interactive use and the *new* command line interface can be found here: https://autoresearch.github.io/autora-workflow/

... and installed from here: https://pypi.org/project/autora-workflow/

It's still not in its final form. To-do's include:

- [x] Document how to run it using Cylc and PIP (rather than conda)
- [ ] Getting the documentation working (including "snippets" of code) when merged into the main repo (at the moment the snippets don't resolve properly)
- [ ] Improve saving and loading – add other options beyond the `.dill` file

Nevertheless, some feedback would be appreciated! Thanks all!

@younesStrittmatter @musslick @gtdang

**Comment by musslick** [comment]
Exciting!! Eager to try it out!
